core/src/wallet/ethereum/explorers/api/EthereumLikeTransactionParser.cpp

```diff
@@ -216,10 +217,13 @@ namespace ledger {
                     std::istringstream issNonce(value);
                     issNonce >> _transaction->nonce;
                 } else if (_lastKey == "input") {
-                    if (isInternalTx) {
-                        _transaction->internalTransactions.back().inputData = fromStringToBytes(value);
-                    } else {
-                        _transaction->inputData = fromStringToBytes(value);
+                    auto inputData = fromStringToBytes(value);
```

Comment:
The constness guy has something to say : "Put const qualifier in front of that variable" :trollface:

Another suggestion is you can avoid to copy the `inputData` by using move semantic.